### ISSUE TITLE
global features

### ISSUE THREAD

**Issue by d04922013** [deepmind/graph_nets#12]
Is there any document or demo showing how to use "global features"? Particularly, how to use it in a loss function such as:

```
def create_loss_ops(target_op, output_ops):
  loss_ops = [
      tf.losses.softmax_cross_entropy(target_op.nodes, output_op.nodes) +
      tf.losses.softmax_cross_entropy(target_op.edges, output_op.edges)
      for output_op in output_ops
  ]
  return loss_ops

```

**Comment by d04922013** [comment]
helpful & thanks
